6.4 Finalization.md

```diff
@@ -1,3 +1,16 @@
+---
+description: As every node might have slightly different perceptions of the Tangle at a given time, so a notion of Finality is necessary to guarantee consensus on the Tangle and its ledger state.
```

Comment:
```suggestion
description: Since every node might have a slightly different perception of the Tangle at a given time, a notion of Finality is necessary to guarantee consensus on the Tangle and its ledger state.
```